src/spasht/ui/template/agent_spasht_contribute.html.twig

```diff
@@ -0,0 +1,102 @@
+{% if pageNo == "show_upload_table" %}
+  <div class="spasht-full-row">
+    <table id="detailsTable" class="simpleTable detailsTable left-align-cells">
+      <tbody>
+        <tr class="detailsRow">
+          <th>Namespace</th>
+          <td>
+            <span id="contributeNamespace">{{ body.namespace }}</span>
+          </td>
+        </tr>
+        <tr class="detailsRow">
+          <th>Name</th>
+          <td>
+            <span id="contributeName">{{ body.name }}</span>
+          </td>
+        </tr>
+        <tr class="detailsRow">
+          <th>Revision</th>
+          <td>
+            <span id="contributeRevision">{{ body.revision }}</span>
+          </td>
+        </tr>
+        <tr class="detailsRow">
+          <th>Type</th>
+          <td>
+            <span id="contributeType"> {{ body.type }}</span>
+          </td>
+        </tr>
+        <tr class="detailsRow">
+          <th>Provider</th>
+          <td>
+            <span id="contributeProvider">{{ body.provider }}</span>
+          </td>
+        </tr>
+        <tr class="detailsRow">
+          <th>status</th>
+          <td>
+            <span>{{ body_menu }}</span>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  {# contribution details form #}
+  <div class="spasht-full-row">
+    <form name="contributeForm" id="contributeForm" method="post" action="">
+      <table>
+        <tr>
+          <td>
+            <p style="text-align:left">Type</p>
+            <select id="ContributionType" style="width:100%">
+              <option value="select" disabled selected>Select</option>
+              <option value="Missing">Missing</option>
+              <option value="Incorrect">Incorrect</option>
+              <option value="Incomplete">Incomplete</option>
+              <option value="Ambigious">Ambigious</option>
+              <option value="Other">Other</option>
+            </select>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p style="text-align:left">Title</p>
+            <input type="text" id="ContributionTitle" style="width:100%"/>
```

Comment:
Use [required attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#required) to allow user to understand it is a required field.